example/newfilewithappauth/main.go

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Please delete this line and switch to per-line comments instead (`//`) for consistency with the rest of this repo.